src/main/java/pages/mainPage/MainPageProduct.java

```diff
@@ -0,0 +1,51 @@
+package pages.mainPage;
+
+import lombok.Getter;
+import models.ProductModel;
+import org.openqa.selenium.By;
+import pages.BasePage;
+
+import java.math.BigDecimal;
+
+@Getter
+public class MainPageProduct {
+    protected BasePage basePage;
+    private final String productXpath;
+
+    public MainPageProduct(int productIndex, String sectionXpath) {
+        this.basePage = new BasePage();
+        this.productXpath = sectionXpath + String.format("//div[@class='thumbnails list-inline']/div[%s]", productIndex);
+        System.out.println(productXpath);
```

Comment:
Delete ``System.out.println(productXpath);``. I guess there is something left after debugging? 🙂